src/server/pps/pps.go

```diff
@@ -4,6 +4,10 @@ import (
 	"google.golang.org/grpc"
 )
 
+const (
+	JobDataPath = "/var/pachyderm/job-data"
```

Comment:
I don't think this will pass the linter (and will fail CI) - I think any exported const's need a comment above explaining them